src/org/usfirst/frc4904/robot/Robot.java

```diff
@@ -72,6 +76,13 @@ public void autonomousExecute() {}
 	@Override
 	public void alwaysExecute() {
 		putSDSubsystemSummary();
+		try {
+			SmartDashboard.putNumber("Boiler Angle Error", RobotMap.Component.lidar.getLidarSensor2().readSensor()[0]);
```

Comment:
There should definitely be a convenience function on LIDAR for different pieces of data